en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim.md

```diff
@@ -39,8 +50,7 @@ Let's start Choreo Connect with a WSO2 API Manager instance which will be acting
     docker ps | grep choreo-connect-
     ```
 
-!!! important
-    The **docker-compose** files referred in this guide are to deploy API Manager with basic configurations. In order to deploy WSO2 API Manager in production grade, use the Docker setup artifacts from [API Manager page](https://wso2.com/api-management/). The docker-compose scripts provided with the distribution are only for trying out purposes.
+    {!includes/deploy/cc-tryout-in-arm64-docker-note.md!}
```

Comment:
Having this in the previous location absorbs the content below it for some reason. Could not find a way to avoid it therefore changed the location of this line. This line does not affect the headings right afterwards.